### ISSUE TITLE
Thoracic lordosis

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10130]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
PMID: 30195269  "All three patients with severe thoracolumbar kyphosis had **thoracic lordosis** with a wedge-like vertebral deformity and anterior vertebral osteophyte at the thoracolumbar junction"

 PMID: 29456757  "All had significant spinal abnormalities, including severe progressive  **thoracic lordosis**, thoracolumbar kyphosis and scoliosis. All had undergone spinal fusion or were being evaluated for surgery."

PMID: 31772430 "**Thoracic lordosis** was associated to scoliosis in 16 patients (16/21; 76.19%), and in six of these (6/16; 37.5%), it was associated to lumbar kyphosis that in two cases was associated with scoliosis (2/21; 9.52%"

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Musculocontractural Ehlers-Danlos syndrome ORPHA:2953
Coffin-Lowry syndrome ORPHA:192 
Marfan syndrome ORPHA:558 
**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6000131
- label: Thoracic lordosis
- parents: [Abnormal curvature of the vertebral column (HP:0010674)](https://hpo.jax.org/app/browse/term/HP:0010674)
- pmid: PMID:30195269
- def: Thoracic lordosis refers to an abnormal curvature of the thoracic spine in which the thoracic spine displays lordosis (inward curve) instead of the normal kyphosis (outward curve).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10130
- synonyms:

**Comment by pnrobinson** [comment]
@carolina-f 
Sorry there was an error
This is the correct term id
HP:0430043
